build.cmd

```diff
@@ -18,6 +18,12 @@ if %errorlevel% equ 0 (
   set outloop=true
 )
 
+echo %* | findstr /i /C:"FloatingTestRuntimeDependencies=true"  1>nul
+if %errorlevel% equ 0 (
+  set TestFixedRuntimeProjectJson=%~dp0src\Common\test-runtime\LatestDependencies\project.json
+  set defaultBuildConfig=/p:Configuration=Windows_NT_Debug
```

Comment:
I think ""Build and test on Linux" be part of build.sh and not build.cmd - maybe that's acceptable here to default to Windows_Debug.